AlphaWallet/Market/Coordinators/UniversalLinkCoordinator.swift

```diff
@@ -310,6 +328,25 @@ class UniversalLinkCoordinator: Coordinator {
         }
     }
 
+    private func checkIfLinkClaimed(r: String, completionHandler: @escaping (Bool) -> Void) {
+        let parameters: Parameters = [ "r": r ]
+        Alamofire.request(
+                Constants.paymentServerClaimedToken,
+                method: .get,
+                parameters: parameters
+        ).responseJSON { result in
+            if let response = result.response {
+                if response.statusCode == 208 || response.statusCode > 299 {
```

Comment:
Does it generate a response code of `208` if it's true?

And this part looks like it should be omitted:

```
|| response.statusCode > 299
```

Reply:
@hboon yes and the second check is for if there is an error, if there is an error (above 299) then we assume we cannot proceed.